DockerProdSetupGuideForMacOS.md

```diff
@@ -146,7 +165,13 @@ or
     ```bash
     source .bash_profile
     ```
+Note: How you set environment and path variables may vary depending on your operating system and terminal used, for instance for zsh on MacOS you typically need to modify .zshrc instead of .bash_profile. To figure out how to set environment variables for your system, consult the guides below or google `how to permentaly set environment/path variables on [insert operating system] [insert terminal type]`.
 
+    For more information on how to set environment variables consult these following guides:
+
+    - https://chlee.co/how-to-setup-environment-variables-for-windows-mac-and-linux/
+    - https://www3.ntu.edu.sg/home/ehchua/programming/howto/Environment_Variables.html
+    - https://unix.stackexchange.com/questions/117467/how-to-permanently-set-environmental-variables
```

Comment:
Same as mentioned above, since this is repeated it will be annoying to maintain edits moving forward - put this in one section and then refer to it in following sections with markdown links.